src/main/java/org/cryptomator/launcher/Cryptomator.java

```diff
@@ -54,6 +54,21 @@
 	}
 
 	public static void main(String[] args) {
+		var printVersion = Optional.ofNullable(args) //
+				.stream() //Streams either one element (the args-array) or zero elements
+				.flatMap(Arrays::stream) //
+				.anyMatch(arg -> "-v".equals(arg) || "--version".equals(arg));
+
+		if(printVersion) {
+			var appVer = Optional.ofNullable(System.getProperty("cryptomator.appVersion"));
+			var buildNumber = Optional.ofNullable(System.getProperty("cryptomator.buildNumber"));
```

Comment:
You can pass a fallback value directly to `System.getProperty("cryptomator.appVersion", "SNAPSHOT")`

Reply:
Resolved by bcb2b088.